spec/defines/image_windows_spec.rb

```diff
@@ -9,42 +9,45 @@
       :kernelrelease             => '10.0.14393',
       :operatingsystemrelease    => '2016',
       :operatingsystemmajrelease => '2016',
+      :program_data_path         => 'C:/ProgramData',
+      :program_files_path        => 'C:/Program Files',
+      :systemroot                => 'C:/Windows',
       :os                        => { :family => 'windows', :name => 'windows', :release => { :major => '2016', :full => '2016' } }
   } }
   context 'with ensure => present' do
       let(:params) { { 'ensure' => 'present' } }
       it { should contain_file('C:/Windows/Temp/update_docker_image.ps1') }
-      it { should contain_exec('& C:/Windows/Temp/update_docker_image.ps1 base') }
+      it { should contain_exec('& C:/Windows/Temp/update_docker_image.ps1 -DockerImage base') }
   end
 
   context 'with docker_file => Dockerfile' do
       let(:params) { { 'docker_file' => 'Dockerfile' }}
-      it { should contain_exec('Get-Content Dockerfile | docker build -t base -') }
+      it { should contain_exec('Get-Content Dockerfile -Raw | docker build -t base -') }
     end
 
     context 'with ensure => present and docker_file => Dockerfile' do
         let(:params) { { 'ensure' => 'present', 'docker_file' => 'Dockerfile' } }
-        it { should contain_exec('Get-Content Dockerfile | docker build -t base -') }
+        it { should contain_exec('Get-Content Dockerfile -Raw | docker build -t base -') }
     end
 
     context 'with ensure => present and image_tag => nanoserver' do
         let(:params) { { 'ensure' => 'present', 'image_tag' => 'nanoserver' } }
-        it { should contain_exec('& C:/Windows/Temp/update_docker_image.ps1 base:nanoserver') }
+        it { should contain_exec('& C:/Windows/Temp/update_docker_image.ps1 -DockerImage base:nanoserver') }
```

Comment:
These tests seem very brittle for the resource name.  Perhaps a better resource name would make sense?